src/core/mesh/qgsmeshrenderersettings.h

```diff
@@ -98,13 +98,12 @@ class CORE_EXPORT QgsMeshRendererScalarSettings
     {
 
       /**
-       * Use data defined on face centers, do not interpolate to vertices
+       * Does not use interpolation
```

Comment:
Can you elaborate on the comment, what is does for various types of data (for data on vertices, data on faces, data on edges, ...)